css-canvas-medical/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+  <link href="//db.onlinewebfonts.com/c/01236efc229254849efb159099afebb3?family=Binary+ITC+Std" rel="stylesheet"
+    type="text/css" />
+</head>
+<body>
+  <div id="container">
+    <div class="row border-bottom">
+      <div class="fourty-percent-column">
+        <h1 class="logo logo-color">canvas</h1>
+      </div>
+      <div class="sixty-percent-column">
+        <a class="grey-text" href="product-placeholder.index">Product</a>
+        <a class="grey-text" href="implementation-placeholder.index">Implementation</a>
+        <a class="grey-text" href="about-us-placeholder.index">About Us</a>
+        <a class="grey-text" href="blog-placeholder.index">Blog</a>
+        <form action="contact-us-placeholder.index">
+          <button class="white-text contact-us-button">Contact Us</button>
+        </form>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full flex-column">
+        <img class="hero-pic" src="images/hero.jpg" alt="woman smiling">
+        <div class="pic-content-position white-text">
+          <h2 class="fontsize50">The EMR Reimagined for Primary Care</h2>
+          <p>Can't remember the last time you talked with a patient without focusing on the EMR? Canvas lets you
+            rediscover joy in the work you do. Capture meaningful data, identify gaps in care, and move seamlessly
+            between tasks within the platform. You stay present with patients while your whole care team stays
+            up-to-speed. Be a better doctor with Canvas, designed for primary care providers with the help of
+              primary care providers.
+          </p>
+
+          <form action="get-started-placeholder.index">
```

Comment:
you don't need form elements to have buttons, you can remove all of the form tags on this page.